src/CrystalStacker.xml

```diff
@@ -0,0 +1,25 @@
+<SPDX name="CrystalStacker License" identifier="CrystalStacker" osi-approved="false">
+  <urls>
+    <url>https://fedoraproject.org/wiki/Licensing:CrystalStacker?rd=Licensing/CrystalStacker</url>
+  </urls>
+  <license>
+    <title>
+      <p>Crystal Stacker is freeware. This means you can pass copies around freely provided you include this
```

Comment:
agreed that it is not a title. I'd also say that we can simply remove the optional tag altogether